src/adagrams.js

```diff
@@ -1,15 +1,146 @@
-export const drawLetters = () => {
-  // Implement this method for wave 1
-};
+class Adagrams {
+  static drawLetters = () => {
+    // Implement this method for wave 1
+    const letterPool = {
```

Comment:
It would be good to have this defined outside of the function so that it does not have to be recreated each time the function is called.